stl/inc/memory

```diff
@@ -1252,10 +1253,47 @@ private:
     }
 
     template <class _Ty0, class... _Types>
-    friend shared_ptr<_Ty0> make_shared(_Types&&... _Args);
+#if _HAS_CXX20 && defined(__cpp_concepts)

```

Comment:
I believe you can drop the `_HAS_CXX20` as `__cpp_concepts` requires it.

Reply:
Done